### DISCUSSION TITLE
字幕翻译问题

### DISCUSSION THREAD

**Discussion by MrFutureV** [Ayanaminn/N46Whisper#27] [General]
可能是prompt还不够完善或者什么问题，字幕生成过程中会出现以下问题
![QQ截图20230321185342](https://user-images.githubusercontent.com/46783891/226585892-58de14ec-907d-48fc-ae13-243f68ff9f38.png)

**Comment by Ayanaminn** [comment]
正常，模型可能会把prompt当作pattern本身，当某些内容不够精准，缺乏上下文，可能会将prompt一起翻译以提高（模型认为的）准确性，等等原因，现在用的这句prompt是比较一般性的？偶尔会出现这种情况

**Reply by MrFutureV** [reply]
> 正常，模型可能会把prompt当作pattern本身，当某些内容不够精准，缺乏上下文，可能会将prompt一起翻译以提高（模型认为的）准确性，等等原因，现在用的这句prompt是比较一般性的？偶尔会出现这种情况

对应该是这样，以后翻译的研究方向估计就是优化prompt了:D

**Reply by MrFutureV** [reply]
> 正常，模型可能会把prompt当作pattern本身，当某些内容不够精准，缺乏上下文，可能会将prompt一起翻译以提高（模型认为的）准确性，等等原因，现在用的这句prompt是比较一般性的？偶尔会出现这种情况

我可以提供个prompt参考一下：
你是一个专业的字幕翻译员，现在我有一个字幕文件如下，请使用人人影视字幕组的翻译风格以及字幕的特性翻译该文件。
